language_detector/main.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
 
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    globalcount = 0
+    lang = ""
+    
+    for dictionary in languages:
+        
+        count = 0
+        for word in text.split():
+            
+            if word in dictionary['common_words']:
+                count += 1
+                    
```

Comment:
Almost exactly our approach too; create a counter, then add to the counter. I like the idea of a global counter, and if an individual count is bigger than the global, then you replace it. Better than creating separate counts for each language. Great job!